pages/common/jetifier.md

```diff
@@ -0,0 +1,12 @@
+# jetifier
+
+> The Jetifier AndroidX transition tool in npm format, with a react-native compatible style.
```

Comment:
Perhaps this will read better?
```suggestion
> Jetifier AndroidX transition tool in npm format, with a react-native compatible style.
```

I'm not sure about the wording of the `in npm format ` bit here. Could you explain a bit more please?